chemprop/cli/train.py

```diff
@@ -636,6 +639,83 @@ def build_splits(args, format_kwargs, featurization_kwargs):
     return train_data, val_data, test_data
 
 
+def summarize(args, dataset: _MolGraphDatasetMixin) -> tuple[list, list]:
+    if args.task_type in ["regression", "regression-mve", "regression-evidential"]:
+        if isinstance(dataset, MulticomponentDataset):
+            y = dataset.datasets[0].Y
+        else:
+            y = dataset.Y
+        y_mean = np.nanmean(y, axis=0)
+        print("YMEAN", y_mean)
+        y_std = np.nanstd(y, axis=0)
+        y_median = np.nanmedian(y, axis=0)
+        mean_dev_abs = np.abs(y - y_mean)
+        frac_1_sigma = np.sum((mean_dev_abs < y_std), axis=0) / len(y)
+        frac_2_sigma = np.sum((mean_dev_abs < 2 * y_std), axis=0) / len(y)
+
+        column_headers = ["Statistic"] + [f"Value {i}" for i in range(y.shape[1])]
+        table_rows = [
+            ["Num. datapoints"] + [f"{len(y)}" for i in range(y.shape[1])],
```

Comment:
For classification we show the breakdown of how many NaN values there are. I wonder if here we should report `Num. datapoints` as the number of non NaN values, have a separate row for number of NaNs, and then a final row with the total number of datapoints. I'm not sure what would be good to call this final row, maybe just "Total datapoints"?

Reply:
The current version now shows Num. smiles, Num. targets, Num. NAN (should be num smiles - num targets, of course). I think that's a bit more clear. Let me know what you think.